include/readout/models/BinarySearchQueueModel.hpp

```diff
@@ -30,9 +30,8 @@ class BinarySearchQueueModel : public IterableQueueModel<T>
     : IterableQueueModel<T>(size)
   {}
 
-  typename IterableQueueModel<T>::Iterator lower_bound(T& element)
+  typename IterableQueueModel<T>::Iterator lower_bound(T& element, bool /*use_backup_strategy=false*/)
```

Comment:
not backup strategy, but `bool with_errors`.
It's not a backup strategy, right? We could use the binary search all the time, as you said. 
It's more like a different lookup strategy in case there are problems with the data held by the LB.
Maybe we need something more clever "strategy selection" here? For now, the bool will be fine:
`bool lookup_strategy=false`